.nvmrc

```diff
@@ -0,0 +1 @@
+v6.9.5
```

Comment:
Can you update to `v6.10.0`, please?  Didn't realise there was a version bump until tonight 😞